[2021-05-12 11:58:32,262] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task4 2021-05-12T10:39:48.267484+00:00 [queued]>
[2021-05-12 11:58:32,270] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task4 2021-05-12T10:39:48.267484+00:00 [queued]>
[2021-05-12 11:58:32,270] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 11:58:32,270] {taskinstance.py:1069} INFO - Starting attempt 11 of 12
[2021-05-12 11:58:32,270] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 11:58:32,377] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): task4> on 2021-05-12T10:39:48.267484+00:00
[2021-05-12 11:58:32,390] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'status_test', 'task4', '2021-05-12T10:39:48.267484+00:00', '--job-id', '124', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_status.py', '--cfg-path', '/tmp/tmpzdqsykr9', '--error-file', '/tmp/tmphioadkhf']
[2021-05-12 11:58:32,391] {standard_task_runner.py:77} INFO - Job 124: Subtask task4
[2021-05-12 11:58:32,380] {standard_task_runner.py:52} INFO - Started process 2231231 to run task
[2021-05-12 11:58:32,633] {logging_mixin.py:104} INFO - Running <TaskInstance: status_test.task4 2021-05-12T10:39:48.267484+00:00 [running]> on host sdn-k8s-b2-3-3.maas
[2021-05-12 11:58:32,771] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=davidwu5858@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=status_test
AIRFLOW_CTX_TASK_ID=task4
AIRFLOW_CTX_EXECUTION_DATE=2021-05-12T10:39:48.267484+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-12T10:39:48.267484+00:00
[2021-05-12 11:58:32,772] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-12 11:58:32,772] {bash.py:158} INFO - Running command: airflow tasks clear -f status_test -y
[2021-05-12 11:58:32,779] {bash.py:169} INFO - Output:
[2021-05-12 11:58:33,740] {bash.py:173} INFO - [2021-05-12 11:58:33,739] {dagbag.py:451} INFO - Filling up the DagBag from /root/airflow/dags
[2021-05-12 11:58:33,753] {bash.py:173} INFO - /home/ubuntu/venv/lib/python3.8/site-packages/airflow/utils/decorators.py:94 DeprecationWarning: provide_context is deprecated as of 2.0 and is no longer required
[2021-05-12 11:58:34,121] {bash.py:177} INFO - Command exited with return code 0
[2021-05-12 11:58:34,273] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=status_test, task_id=task4, execution_date=20210512T103948, start_date=20210512T115832, end_date=20210512T115834
[2021-05-12 11:58:34,433] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-12 11:58:34,461] {local_task_job.py:146} INFO - Task exited with return code 0
